alonzo/impl/cardano-ledger-alonzo.cabal

```diff
@@ -14,11 +14,29 @@ copyright:           2020 Input Output (Hong Kong) Ltd.
 category:            Network
 build-type:          Simple
 
-library
+common base                         { build-depends: base                             >= 4.12       && < 4.15     }
```

Comment:
Why the super-weird formatting on this line?

Reply:
Haha, yeah it is super-weird.

I copied it from here: https://github.com/input-output-hk/cardano-node/blob/import-cardano-submit-api-from-cardano-rest-repo-4/cardano-submit-api/cardano-submit-api.cabal

It's inspired by what Kmett does in his project: https://github.com/ekmett/coda/blob/master/coda.cabal

Which is to be able to specify constrains of any given package exactly once per cabal file.

Of course, I can put use more conventional syntax for this stanza too if that's preferable.

Reply:
I've pushed a new commit to use conventional formatting on base stanzas.